test/e2e/config/nutanix.yaml

```diff
@@ -260,6 +261,16 @@ variables:
   NUTANIX_PRISM_ELEMENT_CLUSTER_IP: ""
   NUTANIX_PRISM_ELEMENT_CLUSTER_USERNAME: ""
   NUTANIX_PRISM_ELEMENT_CLUSTER_PASSWORD: ""
+  # Note: Following parameters are required for failure domain testing
```

Comment:
can we document the config required to be present for this test to run?

Reply:
Added info in the PR description